include/drivers/astar/astar_driver.h

```diff
@@ -33,13 +33,15 @@ Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
 /* for size-t */
 #ifdef __cplusplus
 #   include <cstddef>
+using General_path_element_t = struct General_path_element_t;
 #else
 #   include <stddef.h>
+typedef struct General_path_element_t General_path_element_t;
 #endif
 
 #include "c_types/pgr_edge_xy_t.h"
 #include "c_types/pgr_combination_t.h"
```

Comment:
Should this be changed to `typedef struct General_path_element_t General_path_element_t` because of #2007?
If yes, I made the change locally and found [these lines](https://github.com/pgRouting/pgrouting/blob/0c83362ad111b6a7a3209a6a4c32691da94535ad/src/bdAstar/bdAstar.c#L45-L46) will need to be changed too.

Reply:
FYI
These ones have not been processed yet:
geom_text_rt.h #1899
pgr_edge_xy_t.h #1911

And I am working on trsp and on combinations :
- https://github.com/cvvergara/pgrouting/commits/issue-1893-refining-try4
- https://github.com/cvvergara/pgrouting/commits/issue-1929-trsp

So anything that has to do with tsp and combinations leave it like it is please

I have no problem with General_path_element_t, as the changes I made on that commit you see were not difficult, more straigh forward than the combinations one.

Reply:
Okay got it.